### ISSUE TITLE
Discovery issue for VAMC "Links must have discernible text" Defect-1 items

### ISSUE THREAD

**Issue by mmiddaugh** [department-of-veterans-affairs/va.gov-team#23791]
## Issue Description

"Links must have discernible text" defects were discovered on several VAMC pages. We need to look at them individually to determine if they all have the same root cause and fix. 

- https://www.va.gov/black-hills-health-care/policies/
- https://www.va.gov/minneapolis-health-care/programs/
- https://www.va.gov/northern-california-health-care/stories/

<details> 
<summary> Policies page </summary>

![image](https://user-images.githubusercontent.com/55411834/116153162-20331c80-a6a4-11eb-9310-c36cedfb1fd2.png)

</details>

<details> 
<summary> Programs page </summary>

![Program page empty jump link](https://user-images.githubusercontent.com/55411834/116153090-0691d500-a6a4-11eb-8ac4-d4abb1719f62.png)

</details>

<details> 
<summary>Stories page has two issues </summary>

![image](https://user-images.githubusercontent.com/55411834/116152981-dba78100-a6a3-11eb-9b72-258a4781e634.png)

![image](https://user-images.githubusercontent.com/55411834/116153022-ecf08d80-a6a3-11eb-911a-a56b7035bf82.png)

</details>

There are other approaches to these issues short/long term, including
- [ ] Help Desk support/training
- [ ] Catching the during CMS scan for broken links/defects

---
## Tasks
- [ ] Dig into code to find source of error (content vs template, etc)
- [ ] For issues which can be addressed from front end, document tasks to resolve

## Acceptance Criteria
- [ ] Defects on the pages listed above have been investigated and a path for resolution is determined. 
---

**Comment by davidconlon** [comment]
these are all content issues.

**Comment by mmiddaugh** [comment]
@1Copenut these defect have been addressed from a content perspective. Can you please add this to your list of issues to validate?

**Comment by 1Copenut** [comment]
@mmiddaugh Confirmed the three listed pages have no more "links must have discernible text errors". Moving this one to closed. Thank you!
